firebase-local/mockData/database_export/csma-blend-default-rtdb.json

```diff
@@ -798,6 +798,6 @@
   },
   "users": {
     "Gcr9kCdvK617pSvK8Vn0uXWSjT0Z": { "private": { "stripeCustomerId": "cus_OSMO1QvLtZ2686" }, "protected": { "organizations": ["0000"] } },
-    "TpyNkRxiWfPwunyYEO9fIXlQ26bc": { "private": { "stripeCustomerId": "jnfsdkjnfsdjlk" }, "protected": { "organizations": ["0000"] } }
+    "TpyNkRxiWfPwunyYEO9fIXlQ26bc": { "protected": { "organizations": ["0000"] } }
```

Comment:
So it was breaking for me often when i was logging into Olive Olive since this stripe cust id doesn't exist in test mode, but i don't even think we need a stripe cust id for this account so i just removed it from the mock data if that's okay